scripts/ncf

```diff
@@ -105,6 +199,21 @@ EOF
 
 # MAIN
 # Simple but enough at the moment
+# Check if a server_version is defined
+if [ "$1" = "-V" ]
```

Comment:
i'm really not convinced by -V to define the version, but I didn't have any better idea